### ISSUE TITLE
Feature request: How round Dinero object without parse to float

### ISSUE THREAD

**Issue by NautilusDev** [dinerojs/dinero.js#638]
Hello! Currently there is no possibility to get a Dinero object rounded from other Dinero object.

Sometimes you need round a Dinero object and use later to other operations.

You could use a code like this
```
// https://v2.dinerojs.com/docs/faq/how-can-i-create-dinero-objects-from-floats
dineroFromFloat({ amount: float, currency, scale }) {
  const factor = currency.base ** currency.exponent || scale;
  const amount = Math.round(float * factor);
  return dinero({ amount, currency, scale });
}

const result = dinero({ amount: 14285714, currency: USD, scale: 6 }); // or by result the a operation
const resultRounded = dineroFromFloat({ amount : toUnit(result, {round: halfUp, digits: 2}), currency: USD, scale: 2 });
toSnapshot(resultRounded);
/*
{
  "amount": 1429,
  "currency": { "code": "USD", "base": 10, "exponent": 2 },
  "scale": 2
}
*/
```

but it is easier to use something like this
```
toRound(result, {round: halfUp, digits: 2}); 
```

**Comment by johnhooks** [comment]
@NautilusDev this is being considered in [PR #309](https://github.com/dinerojs/dinero.js/pull/309)

[transformScale](https://github.com/dinerojs/dinero.js/blob/45b5e671f70c07fc2588613006e6919fc95808d2/packages/core/src/api/transformScale.ts) would have a new divide argument of a `DivideOperation` type. This would allow rounding one dinero object to another without having the use `toUnit` the way you are in the example.

```js
import { USD } from '@dinero.js/currencies';
import { dinero, transformScale, toSnapshot, halfUp } from 'dinero.js'

const d = dinero({ amount: 14285714, currency: USD, scale: 6 });
toSnapshot(transformScale(d, 2, halfUp));
/*
{
  "amount": 1429,
  "currency": { "code": "USD", "base": 10, "exponent": 2 },
  "scale": 2
}
*/
```

**Comment by sarahdayan** [comment]
Hey @NautilusDev, as @johnhooks mentioned this is fixed in [`v2.0.0-alpha.11`](https://github.com/dinerojs/dinero.js/releases/tag/v2.0.0-alpha.11) with the new `divider` you can pass to `transformScale`.

Here's how you can use it: https://codesandbox.io/s/condescending-dewdney-4yiq4v?file=/main.js
And the docs: https://v2.dinerojs.com/docs/api/conversions/transform-scale
